roles/inject_openshift_kube_objects/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: "Create the kube_objects directory"
+  shell: "mkdir -p {{ kube_objects_dir }}"
+
+- name: "Decode the symmetric key received from the user"
+  shell: "echo \"{{ encoded_symm_key }}\" | openssl enc -d -base64 > {{ kube_objects_dir }}/get_sym_key"
+  no_log: true
+  
+- name: "Decrypt the symmetric key using ssh private key"
+  shell: "openssl rsautl -decrypt -oaep -inkey {{ ssh_private_key }} -in {{ kube_objects_dir }}/get_sym_key -out {{ kube_objects_dir }}/secret.key"
+
+- name: "Read the encrypted kube_objects decode and store them in a file"
+  shell: "echo \"{{ kube_objects }}\" | tr \" \" \"\n\" | openssl enc -d -base64 > {{ kube_objects_dir }}/get_kubeObjects.txt"
+  no_log: true
+  
+- name: "Decrypt the kube_objects through the shared symmetric key"
+  shell: "openssl aes-256-cbc -d -in {{ kube_objects_dir }}/get_kubeObjects.txt -out {{ kube_objects_dir }}/kube_objects.yaml -pass file:{{ kube_objects_dir }}/secret.key"
```

Comment:
It would be good to use explicit command here for encryption `enc` and not directly algorithm
`openssl aes-256-cbc` -> `openssl enc -aes-256-cbc` 

There is a warning:
```
sh$ openssl aes-256-cbc -d -in ./enc_kube_objects -out kube_objects.yaml -pass file:./secret.key
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.

sh$ openssl version
OpenSSL 1.1.1d FIPS  10 Sep 2019

sh$ rpm -q openssl
openssl-1.1.1d-2.fc30.x86_64
```

Using deprecated things needn't be best with openssl; because they might remove it later due to security reasons.
We might ignore it or we might change that
`openssl enc -aes-256-cbc` -> `openssl enc -aes-256-cbc -pbkdf2`. But the same parameter would need to be used as part of encryption on RHC side

Reply:
I think we can ignore warning. Because -pbkdf2 is not supported with older openssl which is used on server side.

Reply:
yes that's right